File: submissions/java-archive-binks/6.java
static void f(int c){int t=0;if(c<2000){t=(((c-1)/100)+1)*100;}else if(c<5000) {t=(((c-1)/500)+1)*500;}else{t=((((int)(c*1.1)-250)/500)+1)*500;}int g=t-c;if (g==0||g>1500) {System.out.print("PAID BY CARD");}else if(g%100!=0){System.out.print("KEEP THE CHANGE YOU FILTY ANIMAL");}else{System.out.print("HERE IS "+t);}}

File: submissions/nightmare-builders/6.js
function f(t){Z=100;q=500;r=t<2e3?Math.ceil(t/Z)*Z:t<5e3?Math.ceil(t/q)*q:Math.round(Math.trunc(t*1.1)/q)*q;console.log(r-t>1500||!(t-r)?"PAID BY CARD":(r-t)%Z>0?"KEEP THE CHANGE YOU FILTY ANIMAL":"HERE IS "+r)}